resources/helm-broker/templates/helm-secret.yaml

```diff
@@ -7,3 +8,4 @@ metadata:
   name: helm-secret
   namespace: kyma-system
```

Comment:
You can use `{{ Release.Namespace }}`

Reply:
nice catch, I've removed that entry totally